plat/hisilicon/hikey/platform.mk

```diff
@@ -84,3 +84,21 @@ BL2_SOURCES		+=	drivers/arm/sp804/sp804_delay_timer.c	\
 				plat/hisilicon/hikey/hikey_io_storage.c	\
 				plat/hisilicon/hikey/hisi_dvfs.c	\
 				plat/hisilicon/hikey/hisi_mcu.c
+
+HIKEY_GIC_SOURCES	:=	drivers/arm/gic/common/gic_common.c	\
+				drivers/arm/gic/v2/gicv2_main.c		\
+				drivers/arm/gic/v2/gicv2_helpers.c	\
+				plat/common/plat_gicv2.c
+
+BL31_SOURCES		+=	drivers/arm/cci/cci.c			\
+				lib/cpus/aarch64/cortex_a53.S		\
+				plat/common/aarch64/platform_mp_stack.S	\
```

Comment:
`platform_mp_stack.S` is already added by the generic `bl31.mk` file so you don't need to add it again. This causes a local build error for me.